frontend/app/templates/components/user-component/my-badges.hbs

```diff
@@ -69,21 +71,24 @@
                 <input type="checkbox" class="ui checkbox downloadCheck" id={{badge.download_link}}>
                 {{/ui-popup}}
               </div>
-
             </div>
           </div>
         </div>
         {{/each}}
-        {{/if}}
         <div class="card">
           <a class="" href="/create-badges">
-            {{#if model.length}}
             <img src="/images/add_more.png" style="margin-top: 70px">
-            {{else}}
-            <img src="/images/add_more.png" style="margin-top: 70px; margin-bottom: 90px">
-            {{/if}}
           </a>
         </div>
+        {{else}}
+        <center>
+          <h2>You have'nt created any badges yet!</h2>
```

Comment:
Make it have not , Short forms do not looks good when in a Business website

Reply:
Okay